lib/engines/content_object_store/app/components/content_object_store/content_block_document/index/summary_card_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render "components/summary_card", {
```

Comment:
it looks like the components and index/show views are still there under `content_block_edition` but are they are being used any more?

Reply:
Good post. I needed to remove these and switch the existing tests over to the new Document based ones 🙌